### ISSUE TITLE
Accessibility: can't create a new site/choose a domain if iOS is set to a larger text size

### ISSUE THREAD

**Issue by eduardozulian** [wordpress-mobile/WordPress-iOS#15925]
When you create a new site from the app, we give the option to start with WordPress.com. You first choose a layout and then you move to the next screen, called "Choose a Domain". Users with larger text sizes will be stuck there.

Similar to #15655 but this one is apparently fixed.

### Expected behavior

Users should be able to:

- See the field **Search Domains** in order to type it
- See the results of the domain they search for

### Actual behavior

Depending on the text size, you'll see the **Search Domains** field and can type a domain. However, you can't see the results | On even larger text sizes, you can't even see the **Search Domains** text field.
------------ | -------------
<img src="https://user-images.githubusercontent.com/1136160/108426936-2d064080-721b-11eb-9b5c-f3554aa59c73.PNG"/> | <img src="https://user-images.githubusercontent.com/1136160/108426951-32638b00-721b-11eb-81ab-524e3ce1d374.PNG"/>

In both cases, it's impossible to scroll down.

### Steps to reproduce the behavior
1. On your phone, go to System Settings > Accessibility > Display & Text Size
2. Make sure that **Larger Accessibility Sizes** is checked
3. Increase the font size 
4. Try to scroll down to either see the search results or to simply find the **Search Domains** field

##### Tested on iPhone 7, iOS 14.4, WPiOS 16.6.1

Report: 3746854-zen

**Comment by kduncanwelke** [comment]
I'm going to investigate both this and the linked #15655 as well. 👍🏻
